cmd/bosun/sched/notify.go

```diff
@@ -131,6 +166,30 @@ func (s *Schedule) notify(rh *RunHistory, st *State, n *conf.Notification) {
 	n.Notify(subject.Bytes(), body.Bytes(), s.Conf, string(st.AlertKey()), attachments...)
 }
 
+// utnotify is single notification for N unknown groups into a single notification
+func (s *Schedule) utnotify(groups map[string]expr.AlertKeys, n *conf.Notification) {
+	var total int
+	now := time.Now().UTC()
+	for _, group := range groups {
+		// Don't know what the following line does, just copied from unotify
+		s.Group[now] = group
```

Comment:
Okay, will do that one in another PR though so the changes for this are clear in the merge (don't know how big it will be to remove it)